src/components/TaskList.tsx

```diff
@@ -3,6 +3,7 @@ import { HStack, Box, Img, Link, Stack, Text } from '@chakra-ui/react';
 import fluxonLogo from '../assets/fluxon-logo.svg';
 import { useFetchTaskList } from '../hooks/useFetchTaskList';
 
+
```

Comment:
I wonder if the formatter doesn't work in your IDE

Reply:
> I wonder if the formatter doesn't work in your IDE
I suppose my formater setting are strange, but it seems to work